### ISSUE TITLE
Normalize params for PET images

### ISSUE THREAD

**Issue by linhlpv** [bbbbbbzhou/UCAN#1]
Hi @bbbbbbzhou , thank you for great works.
I am interesting about your works and paper and I have a question about normalize params for PET and MRI images.
Could you explain that how do you choose this params for PET and MRI images, plz?
Thank you so much and hope to hear from you soon.

**Comment by bbbbbbzhou** [comment]
Hi, thank you for your interest in our work.
 Per the data preprocessing details in the paper:
We normalized our data by first dividing the maximal value, and then multiplying it by 2, and then minus 1 to ensure the intensity lies in [−1, 1]. During the evaluation, the data are denormalized.

**Comment by linhlpv** [comment]
Thank you for replying me.
Could you explain more about the max value? This is the global maximal value for all images in training set or just the maximal value in each image.
Thank you.

**Comment by bbbbbbzhou** [comment]
We used maximal value for each individual image. But you could try population max for normalization.

**Comment by linhlpv** [comment]
Thank you so much!
